onward/app/controllers/ContentCardController.scala

```diff
@@ -0,0 +1,55 @@
+package controllers
+
+import com.gu.contentapi.client.model.v1.ItemResponse
+import common.{Edition, ExecutionContexts, JsonComponent, Logging}
+import contentapi.ContentApiClient
+import implicits.Requests
+import layout.ContentCard
+import model.{ApplicationContext, Cached, NoCache}
+import play.api.mvc.{Action, Controller, RequestHeader}
+import play.twirl.api.HtmlFormat
+
+import scala.concurrent.Future
+
+class ContentCardController(contentApiClient: ContentApiClient)(implicit context: ApplicationContext) extends Controller with Paging with Logging with ExecutionContexts with Requests   {
```

Comment:
Good point about the duplication. I wasn't sure the best way to approach this but I've had a go in my most recent commit. Could you take a look and let me know what you think?